### PR TITLE
Release/0.19.0.1

### PR DESCRIPTION
**Pull request by cqc-alec**

### CODE REVIEW

File: modules/pytket-braket/docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.16.0 (February 2022)
+----------------------
+
+* Updated pytket version requirement to 0.19.
```
Comment (cqc-melf) [inline modules/pytket-braket/docs/changelog.rst:7]: Should this include the change in the gate set?

File: modules/pytket-qiskit/docs/changelog.rst [hunk 2]
```diff
@@ -6,6 +6,7 @@ Changelog
 
 * Qiskit version updated to 0.34.
 * Updated pytket version requirement to 0.19.
+* Drop support for Python 3.7; add support for 3.10.
```
Comment (cqc-melf) [inline modules/pytket-qiskit/docs/changelog.rst:9]: What about the 3.10 problem?

File: modules/pytket-qsharp/docs/changelog.rst [hunk 3]
```diff
@@ -6,6 +6,7 @@ Changelog
 
 * Updated qsharp version requirement to 0.22.
 * Updated pytket version requirement to 0.19.
+* Drop support for Python 3.7; add support for 3.10.
```
Comment (cqc-melf) [inline modules/pytket-qsharp/docs/changelog.rst:9]: What about the 3.10 problem?

File: modules/pytket-qulacs/docs/changelog.rst [hunk 4]
```diff
@@ -5,6 +5,7 @@ Changelog
 ----------------------
 
 * Updated pytket version requirement to 0.19.
+* Drop support for Python 3.7; add support for 3.10.
```
Comment (cqc-melf) [inline modules/pytket-qulacs/docs/changelog.rst:8]: What about the 3.10 problem?

File: modules/pytket-qiskit/docs/changelog.rst (hunk 2 above)
Comment (cqc-alec) [reply modules/pytket-qiskit/docs/changelog.rst:9]: This is mentioned in the `intro.rst`,

File: modules/pytket-qsharp/docs/changelog.rst (hunk 3 above)
Comment (cqc-alec) [reply modules/pytket-qsharp/docs/changelog.rst:9]: This is mentioned in the `intro.rst`,

File: modules/pytket-qulacs/docs/changelog.rst (hunk 4 above)
Comment (cqc-alec) [reply modules/pytket-qulacs/docs/changelog.rst:8]: This is mentioned in the `intro.rst`,

File: modules/pytket-braket/docs/changelog.rst (hunk 1 above)
Comment (cqc-alec) [reply modules/pytket-braket/docs/changelog.rst:7]: Will add a note about this.

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**